Syntaxes/Markdown Extended.JSON-tmLanguage

```diff
@@ -42,6 +43,26 @@
             ]
         },
         {
+            "begin": "<%\\s",
+            "end": "%>$",
+            "name": "markup.raw.ruby.erb",
+            "patterns": [
+                {
+                    "include": "source.ruby"
+                }
+            ]
+        },
+        {
+            "begin": "\\{:",
+            "end": "\\}",
+            "name": "markup.raw.markdown-extension.inline.attribute",
+            "patterns": [
```

Comment:
this seems like it could cause issues. Have you had any problems with handlebars templates in the same document? 

I also use static site generators quite a bit, I want to make sure this doesn't cause conflicts. For reference:

- https://github.com/assemble/assemble
- https://github.com/assemble/verb
- https://github.com/jonschlinkert/remarkable